doc/links.txt

```diff
@@ -14,15 +14,15 @@
 
 .. ### Links to Nordic pages
 
-.. _`nRF9160 modem firmware zip file`: https://www.nordicsemi.com/Products/Low-power-cellular-IoT/nRF9160/Download#98C9E2578566420786ABD40B695FDB9B
+.. _`nRF9160 modem firmware zip file`: https://www.nordicsemi.com/Products/Low-power-cellular-IoT/nRF9160/Download#0B34B59935AF4AFCB7AB93E9646C1F53
 
 .. _`nRF9160 modem TLS cipher suites`: https://www.nordicsemi.com/Products/Low-power-cellular-IoT/nRF9160/Download#infotabs
 
 .. ### Other links
 
 .. _`Beej's Guide to Network Programming`: https://beej.us/guide/bgnet/
 
-.. _`ZBOSS API documentation`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss/index.html
+.. _`ZBOSS API documentation`:
```

Comment:
Why is there no link ?

Reply:
Because we don't want to state the same link twice. If there is no link after the definition, it'll use the link from the next line.
So basically, this line and the next are aliases for the same link.